pages/book.vue

```diff
@@ -0,0 +1,215 @@
+<script setup lang="ts">
+definePageMeta({
+  layout: false,
+});
+
+const booklist = [
+  {
+    id: 1,
+    image: "~/assets/images/elon.jpeg",
+    name: "Elon Mask",
+    description: "Ashlee Vance",
+    icon: 3,
+  },
+  {
+    id: 2,
+    image: "~/assets/images/tools.jpeg",
+    name: "Tools of Titans",
+    description: "Team Feriss",
+    icon: 3,
+  },
+  {
+    id: 3,
+    image: "~/assets/images/jen.jpeg",
+    name: "You are a badass",
+    description: "Jen Sincero",
+    icon: 2,
+  },
+];
+</script>
+
+<template>
+  <div class="wrapper">
+    <div class="top-nav">
+      <i class="bi bi-list" style="font-size: 2rem"></i>
+      <h2>Bookstop</h2>
+    </div>
+    <div class="searchbar">
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" />
+        <button class="btn btn-outline-primary" type="submit">Search</button>
+      </form>
+    </div>
+    <div class="pc-tab">
+      <input id="tab1" checked type="radio" name="pct" />
+      <input id="tab2" type="radio" name="pct" />
+      <input id="tab3" type="radio" name="pct" />
+      <nav>
+        <ul>
+          <li class="tab1">
+            <label for="tab1">Explore</label>
+          </li>
+          <li class="tab2">
+            <label for="tab2">Recent Books</label>
+          </li>
+          <li class="tab3">
+            <label for="tab3">New</label>
+          </li>
+        </ul>
+      </nav>
+      <section>
+        <div class="tab1">
+          <ExploreTab />
+        </div>
+        <div class="tab2">
+          <h2>Recent Books</h2>
+          <div v-for="b in booklist" :key="b.id">
+            <BookList :book="b" />
+          </div>
+        </div>
+        <div class="tab3">
+          <NewTab />
+        </div>
+      </section>
+    </div>
+  </div>
+  <NavBottom />
+</template>
+
+<style lang="scss" scoped>
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css");
+@import "assets/sass/main.scss";
+@import "assets/sass/variables";
```

Comment:
Declare `variables` sass file inside nuxt.config becuse it's a common declaration of variables